tendermint/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name       = "tendermint"
-version    = "0.10.0" # Also update `html_root_url` in lib.rs when bumping this
+version    = "0.11.0" # Also update `html_root_url` in lib.rs when bumping this
 license    = "Apache-2.0"
 homepage   = "https://www.tendermint.com/"
 repository = "https://github.com/tendermint/kms/tree/master/crates/tendermint"
```

Huh, didn't I update the repository here? Replace with 
```suggestion
repository = "https://github.com/interchainio/tendermint-rs/tree/master/tendermint"
```